src/rubintv/handlers/external/endpoints.py

```diff
@@ -261,13 +286,20 @@ async def get_historical_day_data(request: web.Request) -> web.Response:
     the_date = date(year, month, day)
     day_dict = historical.get_events_for_date(camera, the_date)
     day_events = flatten_events_dict_into_list(camera, day_dict)
-    grid_columns = build_grid_columns_css(camera.channels)
+
+    metadata_json = "{}"
+    metadata_url = get_metadata_url(bucket.name, camera.slug, date_str)
+    metadata_json = requests.get(metadata_url).text
```

Comment:
Same as above except this time the `text` attribute is accessed without checking the return status.  I think that will raise an exception in the case that the status isn't 200.  This is also another reason to try not to repeat this block, if possible

Reply:
Fixed in the refactor of this as a function now.